src/main/java/com/softserveinc/ita/rozetka/page_objects/RozetkaSearchResultsPage.java

```diff
@@ -0,0 +1,61 @@
+package com.softserveinc.ita.rozetka.page_objects;
+
+import com.codeborne.selenide.ElementsCollection;
+import com.codeborne.selenide.Selenide;
+import com.codeborne.selenide.SelenideElement;
+
+import static com.codeborne.selenide.Selenide.$x;
+
+public class RozetkaSearchResultsPage {
+    public ElementsCollection getSearchResultList() {
+        return Selenide.$$x("//span[@class='goods-tile__title']");
+    }
+
+    public RozetkaProductPage pressProductByNumber(int number) {
```

Comment:
rename to navigateToProductPage() navigateToProductPageByIndex() or navigateToProductPageByNumber()